blog.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+  </head>
+  <body>
+
+
+    <header class="header-img">
+      <div class="b2box">
+        <img class="logohdr adoptadog-logo" src="./images/adoptadog-logo.svg" alt="Company Logo">
+        <h2 class="hdrtxt">Seattle's premiere dog adoption agency</h2>
+      </div>
+        <a class="ckout" href="./checkout.html">Checkout</a>
+
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="./index.html">Home</a>
+        </li>
+        <li>
+          <a href="./dogs.html">Dogs</a>
+        </li>
+        <li>
+          <a href="#">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main class="grid main-content wrapper">
+      <section>
+        <h2>Adoptadog Blog</h2>
+          <article class="row instructors c-box">

```

Comment:
Would suggest using flexbox here to allow wrapping on mobile. Then grouping your h2 & p tag into a separate div. This will allow you to have the text aligned on the right of each blog image.